routes/utils/errors.js

```diff
@@ -47,5 +47,9 @@ function parse (error) {
     error.message = 'Only admins can access /users'
   }
 
+  if (error.message === 'Document update conflict') {
+    error.message = 'An account with that username already exists'
+  }
+
```

Comment:
No, it's ok. Thank you so much for your feedback! I will look into it and try to solve it in `accounts.add` method. But it can take me longer (few days) since I'm abroad and don't have access to my own computer. Is that a problem? I hope not cause I ould really like to finish this and then hopefully contribute some more.